base/system/diskpart/diskpart_msg.mc

```diff
@@ -972,39 +974,41 @@ Example:
     CREATE PARTITION EXTENDED SIZE=1000
 .
 Language=Romanian
-    Creates an extended partition on the disk with focus.
-    Applies to master boot record (MBR) disks only.
+    Crează o partiție extinsă pe discul în cauză.
+    Se aplică doar la discurile cu Registrul principal de inițializare (MBR).
 
-Syntax:  CREATE PARTITION EXTENDED [SIZE=<N>] [OFFSET=<N>] [ALIGN=<N>] [NOERR]
+Sintaxă:  CREATE PARTITION EXTENDED [SIZE=<N>] [OFFSET=<N>] [ALIGN=<N>] [NOERR]
 
-    SIZE=<N>    The size of the partition in megabytes (MB). If no size is
-                given, the partition continues until there is no more free
-                space in the extended partition.
+    SIZE=<N>    Mărimea partiției în megaocteți (Mo). Dacă nu este dată nicio
+                mărime, partiția continuă până când nu mai este spațiu liber
+                în partiția extinsă.
 
-    OFFSET=<N>  The offset, in kilobytes (KB), at which the partition is
-                created. If no offset is given, the partition will start
-                at the beginning of the first free space on the disk that
-                is large enough to hold the new partition.
+    OFFSET=<N>  Decalajul, în kiloocteți (Ko), la care partiția este creată.
+                Dacă nu este dat niciun decalaj, partiția va începe de la
+                primul spațiu liber de pe disc ce este suficient de mare ca
+                să țină noua partiție.
 
-    ALIGN=<N>   Typically used with hardware RAID Logical Unit Number (LUN)
-                arrays to improve performance. The partition offset will be
-                a multiple of <N>. If the OFFSET parameter is specified, it
-                will be rounded to the closest multiple of <N>.
+    ALIGN=<N>   De obicei, folosit cu matrici de dispozitive cu Număr de
+                unitate logică (LUN) RAID pentru a îmbunătăți performanța.
+                Decalajul partiției va fi un multiplu a lui <N>. Dacă este
+                specificat parametrul OFFSET, va fi rotunjit la cel mai
+                apropiat multiplu a lui <N>.
 
-    NOERR       For scripting only. When an error is encountered, DiskPart
-                continues to process commands as if the error did not occur.
-                Without the NOERR parameter, an error causes DiskPart to exit
-                with an error code.
+    NOERR       Doar pentru scriere de fișiere script. Când apare o eroare,
```

Comment:
"For scripting only" gets translated here as "doar pentru scriere de fișiere script", but in other instances as "doar pentru automatizare". Please take care of consistency.